### ISSUE TITLE
AWS AMI availability

### ISSUE THREAD

**Issue by giuliojiang** [HPCE/hpce-2017-cw6#48]
The public AMIs are only available if the selected region in AWS is US East (Virginia), all other regions don't seem to be able to find the public HPCE AMI.

Also searching only lists `HPCE-2017-GPU-Image`, but no `HPCE-2017-GPU-Image-v2`

**Comment by m8pple** [comment]
Yes, they are only available in the one region as it costs money for every new region
that they are made available in, and Virginia tends to have good instance pricing
and availability.

As far as I can tell, the HPCE-2017-GPU-Image-v2 image is public (just double-checked) and
available. Possibly it took a while to propagate through? Usually the AMIs are publically
available within an hour, and it was created this morning.
